### DISCUSSION TITLE
Multiplatform Support

### DISCUSSION THREAD

**Discussion by Lunchb0ne** [AlexvZyl/nordic.nvim#85] [Ideas]
Love how the theme is coming along!

Are there any plans to bring the theme to other editors?

**Comment by AlexvZyl** [comment]
Thanks!

At this stage I mainly use Neovim to do my work, so it is unlikely that I will be the one porting this theme.  

But if I switch editors (which is also unlikely, I love Neovim) I might bring it along.

**Comment by Lunchb0ne** [comment]
If you don't mind then I would really like to port this theme to VSCode and iTerm2. Will try to make a PR when I get time.

**Comment by AlexvZyl** [comment]
Sure!  You are more than welcome.

**Comment by kuntau** [comment]
Did you write all other terminal theme by hand or have converter? I would love to have nordic theme for [wezterm](https://wezfurlong.org/wezterm/)

**Comment by AlexvZyl** [comment]
I wrote the kitty theme by hand, and @lcrownover did the others by hand also.
